src/sentry/relay/config/metric_extraction.py

```diff
@@ -406,11 +406,12 @@ def convert_widget_query_to_metric(
     """
     metrics_specs: list[HashedMetricSpec] = []
 
-    if not widget_query.aggregates:
-        return metrics_specs
+    aggregates = widget_query.aggregates or []
+    groupbys = widget_query.columns or []
 
-    aggregates = widget_query.aggregates
-    groupbys = widget_query.columns
+    # Tables do not have aggregates, however, we create metrics from its columns
+    if not widget_query.aggregates:
+        aggregates, groupbys = _separate_aggregates_and_groupbys_from_columns(groupbys)
```

Comment:
Does this affect table fallback? We still want a fallback until 90d from now when tables are extracting properly.